components/Dashboard/Table/index.js

```diff
@@ -10,7 +10,9 @@ import Paper from "@mui/material/Paper";
 export default function BasicTable({map}) {
 	return (
 		<TableContainer component={Paper}>
-			<Table sx={{minWidth: 350}} aria-label="simple table">
+			<Table
+				sx={{minWidth: 350, minHeight: 115}}
+				aria-label="simple table">
```

Comment:
I think that this would not help me as a screen reader user. What about describing the purpose of the table like "table with ... / about ... / for..." so that the user knows what's happening in there?